lib/rucio/core/replica.py

```diff
@@ -644,16 +648,16 @@ def _resolve_dids(dids, unavailable, ignore_availability, all_states, resolve_ar
                                                                 models.DataIdentifier.name,
                                                                 models.DataIdentifier.did_type,
                                                                 models.DataIdentifier.constituent)\
-                                                         .with_hint(models.DataIdentifier, "INDEX(DIDS DIDS_PK)", 'oracle')\
-                                                         .filter(or_(*did_clause)):
+            .with_hint(models.DataIdentifier, "INDEX(DIDS DIDS_PK)", 'oracle')\
+                .filter(or_(*did_clause)):
             if resolve_archives and constituent:
                 # file is a constituent, resolve to parent archives if necessary
                 archive = session.query(models.ConstituentAssociation.scope,
                                         models.ConstituentAssociation.name)\
-                                 .filter(models.ConstituentAssociation.child_scope == scope,
-                                         models.ConstituentAssociation.child_name == name)\
-                                 .with_hint(models.ConstituentAssociation, "INDEX(ARCHIVE_CONTENTS ARCH_CONTENTS_PK)", 'oracle')\
-                                 .all()
+                    .filter(models.ConstituentAssociation.child_scope == scope,
+                            models.ConstituentAssociation.child_name == name)\
+                    .with_hint(models.ConstituentAssociation, "INDEX(ARCHIVE_CONTENTS ARCH_CONTENTS_PK)", 'oracle')\
+                    .all()
```

Comment:
please revert unrelated changes

Reply:
I'll try... it's hard when yours and mine autoformat differ.